### PR TITLE
Patch 26 (Cutout animation, many changes)

### PR DESCRIPTION
**Pull request by bitbutter**
Edited for clarity and concision.

### CODE REVIEW

**Comment by bitbutter** [comment]
Not sure if the information in this article is all still relevant to 3.1.

File: tutorials/animation/cutout_animation.rst [hunk 1]
```diff
@@ -103,52 +90,42 @@ Sprite:
...
+hierarchy. This node applies its own transform (including any transformation it inherits from its parents) to the remote node it targets.
 
-This enables to have a visibility order independent from the
-hierarchy.
+This allows us to correct the visibility order of our elements independent from the
+locations of those parts in the cutout hierarchy.
 
-Simply create two more nodes as children from torso, remote_arm_l and
-remote_hand_l and link them to the actual sprites:
+Create a `RemoteTransform2D` node as a child of the torso. Call it remote_arm_l. Create another RemoteTransform2D node inside the first and call it remote_hand_l. Use the Remote Path property of the two new nodes to target the arm_l and hand_l sprites respectively:
```
Comment (akien-mga) [inline tutorials/animation/cutout_animation.rst]: Note that you should use double backticks for inline code in .rst, i.e.
<pre>
``RemoteTransform2D``
</pre>
Please also use inline code markup for any variable/node name like `remote_arm_l`.

**Comment by akien-mga** [comment]
Squashed and merged manually as d12a5c81 with some further edits to line wrapping, inline code markup and fixed some typos.

**Comment by akien-mga** [comment]
Thanks for your contribution!

**Pull request closed without merging**